codegen/src/main/java/org/web3j/codegen/SolidityFunctionWrapper.java

```diff
@@ -659,11 +668,20 @@ private static ParameterizedTypeName buildRemoteCall(TypeName typeName) {
                 ClassName.get(RemoteCall.class), typeName);
     }
 
-    private static void buildTransactionFunction(
+    private void buildTransactionFunction(
             AbiDefinition functionDefinition,
             MethodSpec.Builder methodBuilder,
             String inputParams) throws ClassNotFoundException {
 
+        if (functionDefinition.hasOutputs()) {
+            //CHECKSTYLE:OFF
+            reporter.report(String.format(
```

Comment:
Now the correct term to use for constant functions is "view" as per http://solidity.readthedocs.io/en/v0.4.21/contracts.html#view-functions. Please can you update to use view instead of the constant modifier?